### PR TITLE
Added AlmaLinux OS Support

### PR DESCRIPTION
**Pull request by nixjobin**
Added AlmaLinux OS (1:1 binary compatible fork of RHEL) to the Supported Operating System.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -116,7 +116,7 @@ function check_dist() {
 	dist=$(echo "${dist}" | tr '[:upper:]' '[:lower:]')
 
 	case "${dist}" in
-		debian|ubuntu|centos|fedora)
+		debian|ubuntu|centos|fedora|almalinux)
 			echo "OK"
 			;;
 		cloudlinux)
```
Comment (bsnyder788) [inline scripts/install.sh:122]: Since almalinux is not officially supported by DO, can you move almalinux down to this block with cloudlinux ? \\

**Review by bsnyder788 — COMMENTED** [review]
Added one request to move almalinux to the unsupported section.

File: scripts/install.sh (hunk 1 above)
Comment (nixjobin) [reply scripts/install.sh:122]: Sure. Will submit another PR with that change.

**Comment by nixjobin** [comment]
Submitted PR 253

**Pull request closed without merging**